test/JDBC/input/BABEL-3614-vu-prepare.sql

```diff
@@ -0,0 +1,109 @@
+CREATE VIEW BABEL_3614_vu_prepare_v1 as (SELECT TRY_CONVERT(DATETIME2(5), '2017-08-25 13:01:10.1234567'));
```

Comment:
Can you add some tests that check the entire range of datetime2? i.e. dates between 0001 and 1753?